docs/usage.rst

```diff
@@ -6,13 +6,15 @@ To use Python Lunar in a project::
 
 	>>> import pylunar
 
-Location: Boston, MA, USA
+Get moon info by entering the location using DMS coordinates. 
+Example: Location - Wellington, New Zealand
```

Comment:
The numeric coordinates specified in the following line are for Boston not Wellington, so the location text should be Boston, MA, USA like the original. This also matches the time zone adjustment below which is set for Boston and not Wellington.